src/engine/providers/perplexity.ts

```diff
@@ -22,38 +23,25 @@ const Perplexity: Provider = {
   },
 
   getChatCompletion: async (config: ProviderConfig, messages: Message[]) => {
-    const openai = new OpenAI({
+    const api = new OpenAI({
       apiKey: config.apiKey,
       baseURL: 'https://api.perplexity.ai',
```

Comment:
Can we move it to separate const BASE_URL ?

Reply:
About that, sorry if this is somewhat obvious if I read the docs, but is Perplexity API 1-1 compatible with OpenAI's?

Reply:
Pplx use OpenAI sdk with own baseUrl in all of their examples so I think their goal is to be and stay compatible. From practical perspective it works with the api in practice